### DISCUSSION TITLE
Rodar no Linux

### DISCUSSION THREAD

**Discussion by WhebertonFialho** [HashLoad/horse#270] [General]
No exemplo para Criar um daemon do linux nao fica muito claro como fazer, tem algum exemplo? pois preciso rodar o meu projeto no linux

**Comment by viniciussanchez** [comment]
Boa tarde... o que necessariamente não ficou claro lá? A princípio o que muda mesmo é a diretiva de compilação do projeto.

**Reply by WhebertonFialho** [reply]
ao rodar fica a mensagem 
"Server is runing on 0.0.0.0:9003
Press return to stop ..."

Mesmo colocando "&" no final do comando para ser executado em segundo plano e nao starta como um servico mesmo colocando no systemctl

**Comment by amaurybsouza** [comment]
A daemon is just a process in the background. If you want to start your program when the OS boots, on linux, you add your start command to /etc/rc.d/rc.local (run after all other scripts) or /etc/startup.sh
